packages/components/src/components/optgroup/optgroup-styles.ts

```diff
@@ -0,0 +1,57 @@
+import type { Theme } from '../../types';
+import { getCss } from '../../utils';
+import {
+  addImportantToEachRule,
+  colorSchemeStyles,
+  getThemedColors,
+  hostHiddenStyles,
+  prefersColorSchemeDarkMediaQuery,
+} from '../../styles';
+import { fontSizeTextXSmall, fontWeightSemiBold, spacingStaticSmall } from '@porsche-design-system/styles';
+import type { Styles } from 'jss';
+import { cssVariableSelectPaddingLeft } from '../select/select-option/select-option-styles';
+import { cssVariableMultiSelectPaddingLeft } from '../multi-select/multi-select-option/multi-select-option-styles';
+
+export const getComponentCss = (isDisabled: boolean, theme: Theme): string =>
+  getCss({
+    '@global': {
+      ':host': addImportantToEachRule({
+        ...colorSchemeStyles,
+        ...hostHiddenStyles,
+      }),
+      '::slotted(*)': {
+        [cssVariableSelectPaddingLeft]: '28px',
```

Comment:
`cssVariableSelectPaddingInlineStart` would be more correct in terms of naming and rtl support